services/image-puller/Dockerfile

```diff
@@ -0,0 +1,78 @@
+ARG PYTHON_VERSION="3.9.12"
+FROM python:${PYTHON_VERSION}-slim-buster as base
+
+LABEL maintainer=GitHK
+
+# simcore-user uid=8004(scu) gid=8004(scu) groups=8004(scu)
+ENV SC_USER_ID=8004 \
+  SC_USER_NAME=scu \
+  SC_BUILD_TARGET=base \
+  SC_BOOT_MODE=default
+
+RUN adduser \
+  --uid ${SC_USER_ID} \
+  --disabled-password \
+  --gecos "" \
+  --shell /bin/sh \
+  --home /home/${SC_USER_NAME} \
+  ${SC_USER_NAME}
+
+
+# Sets utf-8 encoding for Python et al
+ENV LANG=C.UTF-8
+# Turns off writing .pyc files; superfluous on an ephemeral container.
+ENV PYTHONDONTWRITEBYTECODE=1 \
+  VIRTUAL_ENV=/home/scu/.venv
+# Ensures that the python and pip executables used
+# in the image will be those from our virtualenv.
+ENV PATH="${VIRTUAL_ENV}/bin:$PATH"
+
+
+# --------------------------------------------
+FROM base as build
+
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+  build-essential \
+  git \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+
+
+# NOTE: python virtualenv is used here such that installed packages may be moved to production image easily by copying the venv
+RUN python -m venv "${VIRTUAL_ENV}"
+
+RUN pip --no-cache-dir install --upgrade \
+  pip~=22.0  \
+  wheel \
+  setuptools
+
+WORKDIR /build
+
+# install this packages's source code
+COPY  --chown=scu:scu image-puller/ .
+RUN pip --no-cache-dir install .
+
+
+
+# --------------------------------------------
+FROM base as production
+
+ENV PYTHONOPTIMIZE=TRUE
+
+# testing defaults
+ENV POSTGRES_USER=scu \
+  POSTGRES_PASSWORD=adminadmin \
```

Comment:
- This is not the right place for this type of envs . Normally we pass them via the docker-compose
- This service does not use pg right?

Reply:
I am still not done with this `Dockerfile`. I still do not use it